base/math.jl

```diff
@@ -124,6 +124,23 @@ for f in (:sinh, :cosh, :tanh, :atan, :asinh, :exp, :erf, :erfc, :expm1)
     @eval ($f)(x::AbstractFloat) = error("not implemented for ", typeof(x))
 end
 
+#functions with special cases for integer arguements
+@inline function exp2(x::Integer)
+	if x > 1023
```

Comment:
I'm sorry.  I did check that file. But apparently, I fail at using the `retab` command in vim.